package.json

```diff
@@ -71,6 +77,8 @@
     "play:build": "nr -C playground build"
   },
   "peerDependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
I guess we are not referencing react-dom?

Reply:
I added this because we are rendering the DOM with components, which is not supported in the React Native platform. This package is not cross-platform.

Reply:
No, that's a different thing. We don't introduce extra deps to "restrict" library usage. It doesn't work on react-native doesn't mean it only work with react-dom, ppl could have different dom implementation if they want

Reply:
Got it, removed it